wasm/src/tests.rs

```diff
@@ -310,34 +301,35 @@ fn test_runtime_error_related_spans() {
                 }
             }
         "#
-        },
-        "",
+    };
+    run_internal(
+        SourceMap::new([("test.qs".into(), code.into())], None),
         |s| output.push(s.to_string()),
         1,
     )
     .expect("code should compile and run");
-    expect![[r#"{"result":{"code":"Qsc.Eval.ReleasedQubitNotZero","end_pos":89,"message":"runtime error: Qubit0 released while not in |0⟩ state\n\nhelp: qubits should be returned to the |0⟩ state before being released to satisfy the assumption that allocated qubits start in the |0⟩ state","related":[{"end_pos":89,"message":"Qubit0","source":"code","start_pos":73}],"severity":"error","start_pos":73},"success":false,"type":"Result"}"#]]
+    expect![[r#"{"result":{"code":"Qsc.Eval.ReleasedQubitNotZero","end_pos":89,"message":"runtime error: Qubit0 released while not in |0⟩ state\n\nhelp: qubits should be returned to the |0⟩ state before being released to satisfy the assumption that allocated qubits start in the |0⟩ state","related":[{"end_pos":89,"message":"Qubit0","source":"test.qs","start_pos":73}],"severity":"error","start_pos":73},"success":false,"type":"Result"}"#]]
     .assert_eq(&output.join("\n"));
 }
 
 #[test]
 fn test_runtime_error_default_span() {
     let mut output = Vec::new();
-    run_internal(
-        indoc! {r#"
+    let code = indoc! {r#"
             namespace Test {
                 @EntryPoint()
                 operation Main() : Unit {
                     use qs = Qubit[-1];
                 }
             }
         "#
-        },
-        "",
+    };
+    run_internal(
+        SourceMap::new([("test.qs".into(), code.into())], None),
         |s| output.push(s.to_string()),
         1,
     )
     .expect("code should compile and run");
-    expect![[r#"{"result":{"code":"Qsc.Eval.UserFail","end_pos":1,"message":"runtime error: program failed: Cannot allocate qubit array with a negative length","related":[{"end_pos":429,"message":"explicit fail","source":"core/qir.qs","start_pos":372}],"severity":"error","start_pos":0},"success":false,"type":"Result"}"#]]
+    expect![[r#"{"result":{"code":"Qsc.Eval.UserFail","end_pos":429,"message":"runtime error: program failed: Cannot allocate qubit array with a negative length","related":[{"end_pos":429,"message":"explicit fail","source":"core/qir.qs","start_pos":372}],"severity":"error","start_pos":372},"success":false,"type":"Result"}"#]]
```

Comment:
These positions seem off to me. I think I know what to do to map source map offsets, but I need to look for a bit